### PR TITLE
Add facet links pathless schema config

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/0wiO10tC/16-email-trigger-mechanism-for-new-content-tagger-process

Facet groups, facets and facet values don't have base paths so add them to the appropriate config in the schema generator.

### CODE REVIEW

File: lib/schema_generator/format.rb [hunk 1]
```diff
@@ -267,7 +267,7 @@ def determined_definition
 
     class Links
       ALLOWED_KEYS = %w(description required minItems maxItems).freeze
-      LINKS_WITHOUT_BASE_PATHS = %w(ordered_contacts ordered_foi_contacts world_locations).freeze
+      LINKS_WITHOUT_BASE_PATHS = %w(facet_groups facet_values ordered_contacts ordered_foi_contacts world_locations).freeze
```
Comment (leenagupte) [inline lib/schema_generator/format.rb]: Shouldn't `facet` be in this list as well?

Comment (leenagupte) [inline lib/schema_generator/format.rb]: Shouldn't `facet` be in this list as well?

Comment (steventux) [reply lib/schema_generator/format.rb]: @leenagupte good point - added.

**Comment by steventux** [comment]
This isn't necessary any more

**Pull request closed without merging**